backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/migrations/V1_RemoveEmailIndex.kt

```diff
@@ -0,0 +1,16 @@
+package app.ehrenamtskarte.backend.migration.migrations
+
+import app.ehrenamtskarte.backend.migration.Migration
+import org.jetbrains.exposed.sql.transactions.TransactionManager
+import org.jetbrains.exposed.sql.transactions.transaction
+
+internal class V1_RemoveEmailIndex : Migration() {
+    val indexName = "email_lower_idx"
+
+    override fun run() {
+        transaction {
+            val dropStatement = "drop index $indexName"
```

Comment:
when i run the migration script the index should be removed, right?
Maybe i started sth wrong :D
`run --args="migrate"` not sure if thats fine

Reply:
jep that should have been fine. Maybe you can try again with the changes @michael-markl applied. Three migrations should be executed here.